package.json

```diff
@@ -136,6 +137,7 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
+    "lodash": "^4.17.21",
```

Comment:
Lodash is a very heavyweight module, have we checked what this does to the bundle size? Even if it's negligible, we've already got a `capitaliseWords` function in `format.ts` so we can just use that